src-electron/generator/helper-session.js

```diff
@@ -115,7 +119,10 @@ function user_endpoints(options) {
  */
 async function user_device_types(options) {
   let promise = queryDeviceType
-    .selectDeviceTypesByEndpointTypeId(this.global.db, this.endpointTypeId)
```

Comment:
Just update `selectDeviceTypesByEndpointTypeId` with composition. All existing code will automatically pick up what you added that way wherever this function is being called.

Reply:
yeah , I ended up completely redoing this PR and forgot about that